content/en/docs/contribute/style/style-guide.md

```diff
@@ -46,20 +46,24 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 When you refer specifically to interacting with an API object, use [UpperCamelCase](https://en.wikipedia.org/wiki/Camel_case), also known as Pascal Case. When you are generally discussing an API object, use [sentence-style capitalization](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).
 
+You may use the word "resource", "API", or "object" to clarify a Kubernetes resource type in a sentence. Note that Kubernetes resource names are capitalized. However, capitalization alone is not a reliable way to identify Kubernetes resource types in a sentence. 
+
+The sentence "However, pods may have multiple volume mounts" describes general concepts, and uses lower case. In contrast, the sentence "The Pod object includes a Volumes field" is talking about API objects, and uses UpperCamelCase. You may also use [Code Style](#code-style) to distinguish API objects. For example, "The `Pod` object includes a `volumes` field."
+
+Capitalization varies in the Kubernetes API. However, API objects are documented in UpperCamelCase. For example, you may see "configMap" in the [API reference](https://kubernetes.io/docs/reference/). It's fine to call that "ConfigMap" instead when writing general documentation. 
+
 Don't split the API object name into separate words. For example, use
 PodTemplateList, not Pod Template List.
 
-Refer to API objects without saying "object," unless omitting "object"
-leads to an awkward construction.
-
 {{< table caption = "Do and Don't - API objects" >}}
 Do | Don't
 :--| :-----
-The pod has two containers. | The Pod has two containers.
-The HorizontalPodAutoscaler is responsible for ... | The HorizontalPodAutoscaler object is responsible for ...
 A PodList is a list of pods. | A Pod List is a list of pods.
-The two ContainerPorts ... | The two ContainerPort objects ...
-The two ContainerStateTerminated objects ... | The two ContainerStateTerminateds ...
+HostPath is a type of Volume. | Hostpath is a type of volume.
+`HostPath` is a type of Volume. | Hostpath is a type of volume.
```

Comment:
I'd write:
```suggestion
`HostPath` is a type of volume. | Hostpath is a type of volume.
```

and if I saw a page that read “`HostPath` is a type of Volume”, I'd probably raise an issue about fixing that. People never `kubectl create volume` whereas they might `kubectl create deployment`, so “Deployment” gets UpperCamelCase but “volume” usually doesn't. That's my rule of thumb for deciding what capitalization to apply: can you interact with it via `kubectl`? There are exceptions but they are rare.

Reply:
:+1: